slidedeck/src/slides/00-intro/01-intro.pug

```diff
@@ -51,15 +50,16 @@ section
       li Project Setup and Build Tools
       li React Basics
       li Routing
-      li Redux
```

Comment:
I think we should leave Redux in here since there is slide content on it, we can just mention that we won't be covering it in the one-day session